controllers/rcj_soccer_referee_supervisor/referee/json_logger.py

```diff
@@ -0,0 +1,41 @@
+import json
+from pathlib import Path
+from datetime import datetime
+from typing import Optional
+
+
+class JSONLogger(object):
+    def __init__(self,
+                 logfile: Path):
+
+        self.logfile = logfile
+
+    def event(self,
+              supervisor,  # Supervisor from supervisor.py
```

Comment:
@Adman this is total craziness -- I could not get the type here since there was a circular dependency. I am not sure what's the correct way of dealing with this.

Would you happen to know?

Reply:
Try this one
```python
from typing import TYPE_CHECKING

if TYPE_CHECKING
    import Supervisor
...
def event(self, supervisor: 'Supervisor'):
```